model_configs/multilabel_classification/v2.5/bgc_tasknn.jsonnet

```diff
@@ -0,0 +1,155 @@
+                    dataset_metadata.train_file),
+  validation_data_path: (data_dir + '/' + dataset_metadata.dir_name + '/' +
+                         dataset_metadata.validation_file),
+  test_data_path: (data_dir + '/' + dataset_metadata.dir_name + '/' +
+                   dataset_metadata.test_file),
+
+  // Model
+  model: {
+    type: 'multi-label-classification-with-infnet',
+    sampler: {
+      type: 'appending-container',
+      log_key: 'sampler',
+      constituent_samplers: [],
+    },
+    task_nn: {
+      type: 'multi-label-text-classification',
+      feature_network: {
+        text_field_embedder: {
+          token_embedders: {
+            x: {
+              type: 'pretrained_transformer',
+              model_name: transformer_model,
+            },
+          },
+        },
+        seq2vec_encoder: {
+          type: 'bert_pooler',
+          pretrained_model: transformer_model,
+        },
+        final_dropout: task_nn_dropout,
+        feedforward: {
+          input_dim: transformer_dim,
+          num_layers: ff_linear_layers,
+          activations: ([ff_activation for i in std.range(0, ff_linear_layers - 2)] + [ff_activation]),
+          hidden_dims: ff_hidden,
+          dropout: ([task_nn_dropout for i in std.range(0, ff_linear_layers - 2)] + [0]),
+        },
+      },
+      label_embeddings: {
+        embedding_dim: ff_hidden,
+        vocab_namespace: 'labels',
+      },
+    },
+    inference_module: {
+      type: 'multi-label-basic',
+      log_key: 'inference_module',
+      loss_fn: {
+        type: 'multi-label-bce',
+        reduction: 'mean',
+        log_key: 'bce',
+      },
+    },
+    oracle_value_function: { type: 'per-instance-f1', differentiable: false },
+    score_nn: null,  // no score nn for basic tasknn model
+    loss_fn: { type: 'zero' },
+    initializer: {
+      regexes: [
+        //[@'.*_feedforward._linear_layers.0.weight', {type: 'normal'}],
+        [@'.*_linear_layers.*weight', (if std.member(['tanh', 'sigmoid'], ff_activation) then { type: 'xavier_uniform', gain: gain } else { type: 'kaiming_uniform', nonlinearity: 'relu' })],
+        [@'.*linear_layers.*bias', { type: 'zero' }],
+      ],
+    },
+  },
+  data_loader: {
+    shuffle: true,
+    batch_size: 8,  // effective batch size = batch_size*num_gradient_accumulation_steps
+  },
+  trainer: {
+    type: 'gradient_descent_minimax',
+    num_epochs: if test == '1' then 10 else 300,
+    grad_norm: { task_nn: 10.0 },
+    num_gradient_accumulation_steps: 4,  // effective batch size = batch_size*num_gradient_accumulation_steps
+    patience: 5,
+    validation_metric: '+fixed_f1',
+    cuda_device: std.parseInt(cuda_device),
+    learning_rate_schedulers: {
+      task_nn: {
+        type: 'reduce_on_plateau',
+        factor: 0.5,
+        mode: 'max',
+        patience: 2,
+        verbose: true,
+      },
+    },
+    optimizer: {
+      optimizers: {  // have only tasknn optmizer
+        task_nn:
```

Comment:
Drop the score_nn optimizer to not run the update loop corresponding to score_nn.